submissions/emmanuelEsuong/css/styles-mediaQuery.css

```diff
@@ -0,0 +1,264 @@
+/* media-query (tablet)*/
+@media screen and (min-width: 765px) and (max-width: 900px) {
+        /* signUp page */
+    /* formfield */
+    #s-form {
+        border-radius: 25px;
+        border: solid 1px;
+        padding: 35px 20px 20px 20px;
+        border: 2px 2px 2px 2px
+    }
+    /* signupform */
+    #form {
+        overflow: hidden;
+        margin-top: 10px;
+        float: right;
+        margin-right: 5.5cm;
```

Comment:
Use `px`

Reply:
Updated:

Please, I'd like to know why I'm using only px?